### DISCUSSION TITLE
Ability to pop out (undock) a Table Results view

### DISCUSSION THREAD

**Discussion by dannicholls244** [kdbinsidebrains/plugin#47] [Ideas]
A useful feature would be the ability to pop out, (or split multiple) Table Results views. When comparing 2 data sets, can be useful to see 2 or more table results side by side. I'm not sure this is possible in the current version (1.6.2).

Similarly to the existing ability to split the console and table views, ideally the user could drag and drop table views from side to side, to allow for viewing of multiple 'Table Result' tabs at once.

**Comment by smklimenko** [comment]
No DragAndDrop but from 1.7 you can open a tab in separate frame now:

![image](https://user-images.githubusercontent.com/3913096/188566427-0e90da18-fb07-4ba2-ba54-14c7f3de9616.png)

**Reply by dannicholls244** [reply]
That looks great, thanks @smklimenko !
